app/helpers/checklist_tests_helper.rb

```diff
@@ -44,14 +44,10 @@ def available_data_criteria(measure, criteria, original_sdc)
     Hash[dc_hash.sort]
   end
 
-  # A data criteria can be uesd for subtitution if it isn't derived (e.g., Occurrence A of, or birthtime)
-  def substituable_data_criteria?(criteria)
-    if criteria[criteria.keys[0]].definition == 'derived' || criteria[criteria.keys[0]].type == 'derived' ||
-       (criteria[criteria.keys[0]].type == 'characteristic' && criteria[criteria.keys[0]].property == 'birthtime')
-      return true
-    else
-      return false
-    end
+  # A data criteria cannot be used for subtitution if it is derived (e.g., Occurrence A of), or birthtime
+  def unsubstituable_data_criteria?(criteria)
+    cr0 = criteria[criteria.keys[0]]
+    cr0['definition'] == 'derived' || cr0['type'] == 'derived' || (cr0['type'] == 'characteristic' && cr0['property'] == 'birthtime')
```

Comment:
Did you also have to negate this value? Since you changed the method to `unsuitable`